sqlitecluster/SQLiteNode.h

```diff
@@ -144,7 +144,7 @@ class SQLiteNode : public STCPNode {
 
     // Updates the internal state machine; returns true if it wants immediate
     // re-updating.
-    bool update(uint64_t& nextActivity);
+    bool update();
```

Comment:
`nextActivity` is no longer relevant. We always process one command and return. We used to process at least one command and then return, if the timeout was hit.